templates/taxi/driver_list.html

```diff
@@ -8,6 +8,7 @@ <h1>Driver list</h1>
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
           ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver.username == user.username %}(Me){% endif %}
```

Comment:
not a big deal but you can also make comparison by id - to compare integers takes less runtime then to compare strings